data/json/itemgroups/Locations_MapExtras/mall_item_groups.json

```
@@ -211,7 +211,7 @@
       [ "knife_trench", 7 ],
       [ "switchblade", 20 ],
       [ "knife_hunting", 20 ],
-      [ "knife_rambo", 5 ]
+      { "distribution": [ { "group": "knife_rambo_full" }, { "group": "knife_rambo_used" } ], "prob": 5 }
```

```suggestion
      { "group": "knife_rambo_full", "prob": 5 }
```
Unless this is a used knife shop, it shouldn't have anything missing.